### DISCUSSION TITLE
ImportError: cannot import name 'PandasAI' from 'pandasai'

### DISCUSSION THREAD

**Discussion by hemant0011** [sinaptik-ai/pandas-ai#874] [Q&A]
Hi all, I was trying to import PandasAI but getting following error.

ImportError: cannot import name 'PandasAI' from 'pandasai' (/home/emr-notebook/.local/lib/python3.9/site-packages/pandasai/__init__.py)

I am currently using pandasai==1.4.1. I have tried same on pandasai==1.5.14

**Comment by gventuri** [comment]
@hemant0011 you need to import a `SmartDataframe` or a `SmartDatalake`, check out the docs:
https://docs.pandas-ai.com/en/latest/getting-started/

**Reply by amanreddy77** [reply]
> @hemant0011 you need to import a `SmartDataframe` or a `SmartDatalake`, check out the docs: https://docs.pandas-ai.com/en/latest/getting-started/

can you explain me in detail ?
